### PR TITLE
Zadatak3-Meet-landing-page

### PR DESCRIPTION
**Pull request by tatjama**
Git: https://github.com/tatjama/zadatak3-meet-landing-page/tree/develop
Preview: https://zadatak3-meet-landing-page-er77cakkj-tatjana.vercel.app/
Vercel: https://zadatak3-meet-landing-page.vercel.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,49 +15,92 @@
...
             Meet makes it easy to connect with others face-to-face virtually and collaborate across any device.
           </p>
           <div class="hero__btns">
-            <button class="hero__btn--left">Download v1.3</button>
+            <button class="hero__btn--left">Download <span class="hero__span"> v1.3</span></button>
             <button class="hero__btn--right">What is it?</button>
           </div>
+        </div>  
+        <div class="hero__imgs-container">
+          <div class="hero__imgs--right">
```
Comment (ghost) [inline index.html]: Are you familiar with React.js? :)

File: index.html [hunk 2]
```diff
@@ -9,23 +9,52 @@
   <title>Meet landing page</title>
 </head>
 <body>
-  Meet makes it easy to connect with others face-to-face virtually and collaborate across any device.
+  <div class="container">
```
Comment (ghost) [inline index.html]: The whole section must be in a container
Screenshot from 2021-09-21 21-24-33
.

File: index.html (hunk 1 above)
Comment (tatjama) [reply index.html]: Yes, I am. 
I made about ten projects using React.js technology.  
My portfolio was made in React.js
https://my-react-portfolio-tatjana.vercel.app
And within it on the page My Work  are two projects that have also been done in React.js
1. Life in different countries...
2. Play it again, Sam!

File: index.html (hunk 2 above)
Comment (tatjama) [reply index.html]: I think I don't understand you quite well..
My structure is:
open body -> open div container -> header -> main -> footer -> close div container -> close body
When you  say "The whole section must be in a container"
Do you mean that I have to create new container with header and main inside like this:
open body -> open div container -> 
                       open new container -> header -> main -> close new container 
-> footer   ->close div container -> close body
?

Comment (ghost) [reply index.html]: Your structure is good. All you have to do is make a container that will have some width. See here how this colleague did it.
https://exercise-2-git-develop-obrad97.vercel.app/

File: index.html (hunk 1 above)
Comment (ghost) [reply index.html]: Well done! It looks great!

File: index.html (hunk 2 above)
Comment (tatjama) [reply index.html]: Is it OK now?
Screenshot (408)

File: index.html (hunk 1 above)
Comment (tatjama) [reply index.html]: Thank you

File: index.html (hunk 2 above)
Comment (ghost) [reply index.html]: Yes, great!

**Pull request merged**